File: 3 - Printing Quotes/C/main.c
#include<stdio.h>
#include<string.h>

#define STRMAXSIZE 100

int main(void){
    char name[STRMAXSIZE];
    char quote[STRMAXSIZE];

    printf("What is the quote? ");
    fgets(quote, STRMAXSIZE, stdin);
    quote[strlen(quote)-1] = 0;//get rid of \n

    printf("Who said it? ");
    fgets(name, STRMAXSIZE, stdin);
    name[strlen(name)-1] = 0;//get rid of \n

    printf("%s says, \"%s.\"", name, quote);

    return 0;
}

File: 3 - Printing Quotes/Golang/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	fmt.Print("What is the quote? ")
	quote, _ := reader.ReadString('\n')
	quote = strings.TrimSuffix(quote, "\n")
	fmt.Print("Who said it? ")
	name, _ := reader.ReadString('\n')
	name = strings.TrimSuffix(name, "\n")

	fmt.Println(name, "says, \"", quote, ".\"")
}
